### PR TITLE
AP-1242 non passported means results header blocks

### PR DESCRIPTION
**Pull request by stephenrichards**
## Display appropriate header blocks on non-passported means assessment results page

[Link to story](https://dsdmoj.atlassian.net/browse/AP-1242)

- New controller `CapitalIncomeAssessmentResultsController` to display results
- Changed `#overview` method in `CFE::Result` to always use `CCMS::ManualReviewDeterminer` to determine whether or not a manual review is required
- Updated `CCMS::ManualReviewDeterminer` to cater for non-passported applications
- Added partials and translations for `CapitalIncomeAssessmentResultsController` 
- Added new set of mock true layer bank transactions
- Added a new field on `Setting` to say which set of bank transactions to use
- Added new test data for `CFE::V2::Result` in factories
- Added test for `CFE::V2::Result` in factories to ensure test data being set up correctly

## Checklist

Before you ask people to review this PR:

- [ ] Tests and rubocop should be passing: `bundle exec rake`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [ ] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [ ] The PR description should say what you changed and why, with a link to the JIRA story.
- [ ] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/models/cfe/v2/result.rb [hunk 1]
```diff
@@ -7,16 +7,32 @@ def assessment_result
         result_hash[:assessment][:assessment_result]
       end
 
+      def capital_assessment_result
+        capital[:assessment_result]
+      end
+
+      def capital_contribution_required?
```
Comment (willc-work) [inline app/models/cfe/v2/result.rb:14]: Could this method use the one above?
`def capital_contribution_required?
    capital_assessment_result  == 'contribution_required'
 end`

File: app/models/cfe/v2/result.rb [hunk 2]
```diff
@@ -7,16 +7,32 @@ def assessment_result
...
-        result_hash[:assessment][:capital]
+      def income_assessment_result
+        disposable_income[:assessment_result]
       end
 
-      def property
-        capital[:capital_items][:properties]
+      def income_contribution_required?
+        disposable_income[:assessment_result] == 'contribution_required'
+      end
```
Comment (willc-work) [inline app/models/cfe/v2/result.rb:28]: Same as other comment, could this method use ```income_assessment_result``` from the method above?

File: config/locales/en/providers.yml [hunk 3]
```diff
@@ -106,6 +109,47 @@ en:
...
+        detail: 'This is because they have too much disposable capital.'
+      capital_contribution_required:
+        heading: '%{name} may need to pay towards legal aid'
+        details:
+          - 'We’ve calculated that your client should pay %{amount} from their disposable capital.'
+          - 'You still have to provide details of the case before we can grant legal aid.'
+      capital_and_income_contribution_required:
+        heading: '%{name} may need to pay towards legal aid'
+        line_1: "We've calculated that your client should pay:"
+        line_2: You still need to provide details of the case before we can grant legal aid.
```
Comment (willc-work) [inline config/locales/en/providers.yml:137]: this line says ```You still need to...``` other lines use ```You still have to ...```

File: app/models/cfe/v2/result.rb (hunk 1 above)
Comment (stephenrichards) [reply app/models/cfe/v2/result.rb:14]: Good catch

**Pull request merged**